compiler/z/codegen/OMRTreeEvaluator.cpp

```diff
@@ -15780,18 +15780,35 @@ OMR::Z::TreeEvaluator::arraycmpSIMDHelper(TR::Node *node,
    TR::Register * vectorOutputReg = cg->allocateRegister(TR_VRF);
    TR::Register * resultReg = needResultReg ? cg->allocateRegister() : NULL;
 
-   // VLL uses lastByteIndexReg as the highest 0-based index to load, which is length - 1
-   generateRILInstruction(cg, TR::InstOpCode::getSubtractLogicalImmOpCode(), node, lastByteIndexReg, 1);
-   if(needResultReg && isArrayCmpLen)
-      generateRRInstruction(cg, TR::InstOpCode::getLoadRegOpCode(), node, resultReg, lastByteIndexReg);
+   if(needResultReg)
+      {
+      if(isArrayCmpLen)
```

Comment:
I know this is the code that already exists, but as we are in this evaluator, can I request you to make changes in the way those `if-else` blocks are written. You can take a look at the certain other code pieces in the same file, but even if there only one statement, As the statement is bit larger, we should put them in curly braces, to increase readability.

Reply:
I add curly braces to long one line if-else statements.